Cargo.toml

```diff
@@ -39,3 +39,4 @@ walkdir = "2.5.0"
 [patch."https://github.com/stackabletech/operator-rs.git"]
 # stackable-operator = { path = "../operator-rs/crates/stackable-operator" }
 # stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "main" }
+stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feature/cluster-detection-local-kubelet" }
```

Comment:
todo: update this when the feature is released in stackable-operator (after 25.7.0)